packages/graphql-codegen-cli/package.json

```
@@ -89,7 +90,8 @@
   },
   "main": "dist/commonjs/index.js",
   "module": "dist/esnext/index.js",
+  "typings": "dist/index.d.ts",
```

```suggestion
  "typings": "dist/esnext/index.d.ts",
```
I think we forgot that one.